server/src/features/users/users.router.ts

```diff
@@ -0,0 +1,20 @@
+import { Router } from 'express';
+import { Service } from 'typedi';
+
+import { UsersController } from './users.controller';
+
+@Service()
+export class UsersRouter {
+  constructor(private readonly usersController: UsersController) { }
+
+  public initUsersRouter(): Router {
+    const usersRouter = Router();
+
+    usersRouter.get('/', this.usersController.getAll.bind(this.usersController));
```

Comment:
Możesz stworzyć na to ładną abstrakcję. A nawet użyć dziedziczenia albo kompozycji albo inicjować generyczny router i używać jego metod. Wiele różnych podejść zamiast zawsze pisać ręcznie `.bind`